stdlib/public/runtime/ImageInspectionELF.cpp

```diff
@@ -78,6 +79,7 @@ static int iteratePHDRCallback(struct dl_phdr_info *info,
   return 0;
 }
 
+SWIFT_RUNTIME_EXPORT
```

Comment:
They still shouldn't need to be exported since all of the `.a` files are ultimately being linked into the same binary. Visibility only affects an executable or dylib, not .o files. In a static binary, ideally, *everything* should be hidden besides `main`, in order to maximize the power of LTO and dead symbol stripping.